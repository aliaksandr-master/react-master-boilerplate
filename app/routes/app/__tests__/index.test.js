/* eslint-env jest */

import route from '../index';



describe('route app', () => {
  it('attach', () => {
    expect(route).toMatchSnapshot();
  });
});
