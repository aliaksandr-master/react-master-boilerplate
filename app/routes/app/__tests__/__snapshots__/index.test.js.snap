// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`route app attach 1`] = `
Object {
  "createChildRoute": [Function],
}
`;
